index.js

```diff
@@ -0,0 +1,168 @@
+          Ruby on rails
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          css
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          javascript
+        </p>
+      </div>
+      <div class="works__card__btn">
+        <button type="button" class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white" onclick="display(${item.id})">
+          See Project
+        </button>
+      </div>
+    </div>
+  </li>
+    <div class="popup-window" id="element-${item.id}">
+    <div class="popup">
+    <i class="fa fa-times" aria-hidden="true" onclick="undisplay(${item.id})">close-icon</i>
+    <h1>${item.name}</h1>
+    <div class="works__card__subtitle flex">
+          <p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">
+            CANOPY
+          </p>
+          <ul class="works__card__subtitle-ul flex">
+            <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">
+              Back End Dev
+            </li>
+            <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">
+              2015
+            </li>
+          </ul>
+        </div>
+        <div class="popup-image-container">
+        <img src="${item.Image}" alt="tonic project" />
+      </div>
+      <div class="flex-text">
+      <div class="text-section">
+      <p>${item.descrip}</p>
+      </div>
+      <div class="button-section">
+      <div class="works__card__topics flex">
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          html
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          Ruby on rails
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          css
+        </p>
+        <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+          javascript
+        </p>
+      </div>
+      <div class="btn">
+      <button><a href="${item.LiveLink}">See Live</a></button>
+      <button><a href="${item.SourceLink}">See Source</a></button>
+      </div>
+      </div>
+      </div>
+    </div>
+    </div>
+    `).join('');
+
+const display = (index) => {
+  const obj = document.getElementById(`element-${index}`);
+  obj.style.display = 'block';
+  console.log('hi');
+};
+
+const undisplay = (index) => {
+  const obj = document.getElementById(`element-${index}`);
+  obj.style.display = 'none';
+};
+
+// contact form
+const formula = document.querySelector('.formClass');
+function errData(data) {
+  document.getElementById('data').innerHTML = data;
+}
+const email = document.getElementById('contact-email');
+formula.addEventListener('input', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    errData('Please input email in lower case');
+    e.preventDefault();
+    return false;
+  }
+  return true;
+});
```

Comment:
- [ ]  Kindly use the submit event to trigger checking for the email case. This is a project requirement. 😄 
![submit](https://github.com/jerymey/My-portfolio/assets/99656687/0da877cf-e2aa-4dc7-9b22-a1d2c34344f9)